packages/odyssey-react/src/utils/focusHandling.ts

```diff
@@ -40,12 +40,7 @@ function setFocus(elem: OptionalHTMLElement): OptionalHTMLElement {
   const allItems: NodeListOf<HTMLElement> = elem.querySelectorAll(
     FOCUSABLE_ITEMS_SELECTOR
   );
-  const focusableItems = Array.from(allItems).filter((el) => {
-    if (el.tabIndex === -1) {
-      return false;
-    }
-    return true;
-  });
+  const focusableItems = Array.from(allItems).filter((el) => el.tabIndex >= 0);
```

Comment:
Aren't there focusable elements (such as buttons) that don't need an explicit `tabindex` attribute?

Reply:
Feel free to ignore this comment given the relevant code wasn't modified in this PR, but we should probably at some point improve the focusableItems lookup to account for elements that are natively focusable without requiring a tabindex attribute.